index.js

```diff
@@ -23,7 +23,7 @@ app.post('/', (req, res) => {
     if (req.warmer) {
         res.send(`"Warmed": true`);
     }
-    console.log(req);
+    // console.log(req);
```

Comment:
so my personal preference is to remove these if you are done with them, not just comment them out